### PR TITLE
Add Python, cURL and Postman documentation for single file upload

### PR DESCRIPTION
**Pull request by StevenBorg**
## Description
Add necessary documentation to Python, cURL and Postman walkthroughs to highlight the newly added (non-DICOMweb standard) API to upload a single file without needing to use multipart/related

## Related issues
Addresses #349 

## Testing
On a new install of the Medical Imaging Server for DICOM, I ran through cURL, Postman and Python end-to-end

### CODE REVIEW

File: docs/resources/Conformance-as-Postman.postman_collection.json [hunk 1]
```diff
@@ -736,27 +858,27 @@
 	],
 	"variable": [
 		{
-			"id": "3e481f3c-c4bf-462e-8ba5-8e82e4108126",
+			"id": "195d429b-8341-43de-9694-10c686612653",
 			"key": "baseUrl",
-			"value": "http://{service-name}.azurewebsites.net"
+			"value": "http://sjbdicomtest.azurewebsites.net"
```
Comment (brandonpollett) [inline docs/resources/Conformance-as-Postman.postman_collection.json:863]: Is this change intentional?

File: docs/tutorials/use-dicom-web-standard-apis-with-python.md [hunk 2]
```diff
@@ -179,12 +179,21 @@ _Details:_
...
+
+# Hack. Need to open up and read through file and load bytes into memory 
+with open(filepath,'rb') as reader:
+    body = reader.read()
+
+headers = {'Accept':'application/dicom+json', 'Content-Type':'application/dicom'}
+
+url = f'{base_url}/studies'
+response = client.post(url, body, headers=headers, verify=False)
+response  # response should be a 409 Conflict if the file was already uploaded abovin the above request
```
Comment (brandonpollett) [inline docs/tutorials/use-dicom-web-standard-apis-with-python.md:196]: nit: "abovin" -> "in"

**Review by brandonpollett — APPROVED** [review]

File: docs/resources/Conformance-as-Postman.postman_collection.json (hunk 1 above)
Comment (StevenBorg) [reply docs/resources/Conformance-as-Postman.postman_collection.json]: Nope!  Thank you!

**Comment by sven-hoffmann** [comment]
Hi @StevenBorg - you mention, that the new single document API is not in the DICOMweb standard. I disagree with this after a little research. There are old versions of the DICOMweb standard (from 2013) which do not contain this, but the most recent version of the standard does. You can read about this here: http://dicom.nema.org/medical/dicom/current/output/html/part18.html#sect_8.6.1
